src/Features/Core/Portable/NavigateTo/AbstractNavigateToSearchService.SearchResult.cs

```diff
@@ -18,89 +17,117 @@ internal abstract partial class AbstractNavigateToSearchService
     {
         private class SearchResult : INavigateToSearchResult
         {
-            public string Name => DeclaredSymbolInfo.Name;
-            public string AdditionalInformation => _lazyAdditionalInfo.Value;
-
-            public string Summary { get; }
+            public string AdditionalInformation { get; }
             public string Kind { get; }
             public NavigateToMatchKind MatchKind { get; }
-            public INavigableItem NavigableItem { get; }
             public bool IsCaseSensitive { get; }
+            public string Name { get; }
             public ImmutableArray<TextSpan> NameMatchSpans { get; }
+            public string SecondarySort { get; }
+            public string? Summary => null;
 
-            public readonly Document Document;
-            public readonly DeclaredSymbolInfo DeclaredSymbolInfo;
-
-            private readonly Lazy<string> _lazyAdditionalInfo;
```

Comment:
there's no point in being lazy here.  we create these values and then immediately marshall them over the OOP boundary, realizing al lthe state of them.